maven.html

```diff
@@ -19,14 +19,28 @@ <h2>Run HelloWorld using Maven</h2>
     &lt;plugin&gt;
         &lt;groupId&gt;org.openjfx&lt;/groupId&gt;
         &lt;artifactId&gt;javafx-maven-plugin&lt;/artifactId&gt;
-        &lt;version&gt;<span class="JFX_MVN_PLUGIN_VERSION">0.0.1</span>&lt;/version&gt;
+        &lt;version&gt;<span class="JFX_MVN_PLUGIN_VERSION">0.0.3</span>&lt;/version&gt;
         &lt;configuration&gt;
             &lt;mainClass&gt;HelloFX&lt;/mainClass&gt;
         &lt;/configuration&gt;
     &lt;/plugin&gt;
 &lt;/plugins&gt;
 </code></pre>
 
+<p>
+    Add the maven dependencies:
+</p>
+
+<pre><code>
+&lt;dependencies&gt;
+  &lt;dependency&gt;
+    &lt;groupId&gt;org.openjfx&lt;/groupId&gt;
+    &lt;artifactId&gt;javafx-controls&lt;/artifactId&gt;
+    &lt;version&gt;<span class="JFX_VERSION">13</span>&lt;/version&gt;
```

Comment:
Same here, it is ok if we set 11 by default, the value will be set to the latest.